array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,15 @@
+/* exported takeRight */
+// check if number is larger
+// than the length of the array
+// if so - return array
+// if not use slice method to take
+// number(count) of elements from the end of array
+// return the result
+function takeRight(array, count) {
+  if (count > array.length) {
+    return array;
+  } else {
+    array.slice(-count);
```

Comment:
This line is doing work and then immediately discarding it, it should be deleted.